src/main/java/Task.java

```diff
@@ -0,0 +1,68 @@
+/**
+ * Tasks are added by the user for KKBot to store and keep track of.
+ *
+ * @author AkkFiros
+ */
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    /**
+     * Constructor for a task
+     * @param description Description of the task that the user inputs
+     */
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    /**
+     * Obtains and returns the task description
+     * @return Task description
+     */
+    public String getDescription() {
+        return this.description;
+    }
+
+    /**
+     * Obtains and returns the completion status of the task
+     * @return The completion status
+     */
+    public boolean getCompletion() {
```

Comment:
Perhaps this method name should be phrased to sound more like a boolean for example, `isCompleted`